client/src/components/CarouselDots/CarouselDots.js

```diff
@@ -0,0 +1,124 @@
+
+import React, {Component} from 'react';
+import "./CarouselDots.css";
+import ReactDOM from "react-dom";
+
+// Class Based React Component
+class CarouselDots extends Component{
+  constructor(props){
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "CarouselDots",
+      numberOfDots: props.numberOfDots,
+      activeLocation: props.activeLocation,
+      activeDotColor: props.activeDotColor,
+      inactiveDotColor: props.inactiveDotColor,
+      hasBorder: props.hasBorder,
+      borderWeight: props.borderWeight,
```

Comment:
If this data is in `props`, then there's no reason to assign it to `state` unless it's data that is going to change. It doesn't look like any of this data is changing